views/home.ejs

```diff
@@ -1,33 +1,34 @@
 <%- include('./partials/head') %>
 <main>
-  <ul class="hoofdpagina">
-    <%articles.forEach(article => {
-      {%>
-      <li>
+  <section>
+    <ul class="post">
+      <%articles.forEach(article => { {%>
+      <li class="post-content">
         <a href="/artikel/<%=article.slug%>">
-
           <picture>
-            <img src="<%=article.jetpack_featured_media_url%>"
-            alt="artikel afbeelding"
-            width="<%=article.yoast_head_json.og_image[0].width%>" 
-            height="<%=article.yoast_head_json.og_image[0].height%>">
+            <img
+              src="<%=article.jetpack_featured_media_url%>"
+              alt="artikel afbeelding"
+              width="<%=article.yoast_head_json.og_image[0].width%>"
+              height="<%=article.yoast_head_json.og_image[0].height%>"
+            />
           </picture>
           <h1><%-article.title.rendered%></h1>
         </a>
-        
-        <div>
-          <time>
-            <%= String(new Date(article.date).getDate()).padStart(2, '0') %>/<%= String(new Date(article.date).getMonth() + 1).padStart(2, '0') %>/<%= new Date(article.date).getFullYear() %>
-          </time>
 
-          <span><%=article.shares%> keer gedeeld</span>   
-        </div>
+        <time>
+          <%= String(new Date(article.date).getDate()).padStart(2, '0') %>/<%=
+          String(new Date(article.date).getMonth() + 1).padStart(2, '0') %>/<%=
+          new Date(article.date).getFullYear() %>
+        </time>
+
+        <span><%=article.shares%> keer gedeeld</span>
+
         <p><%- article.excerpt.rendered %></p>
       </li>
-      <%}
-    })%>
-  </ul>
-
+      <%} })%>
+    </ul>
+  </section>
   <ul class="categories-home">
     <li>
       <h1>Onze supporters</h1>
```

Comment:
Een h1 in een list-item klinkt niet heel logisch en semantisch. Weet je zeker dat dat optimaal is?